lib/client-assets.php

```diff
@@ -203,6 +203,13 @@ function gutenberg_register_scripts_and_styles() {
 		filemtime( gutenberg_dir_path() . 'build/dom/index.js' ),
 		true
 	);
+	wp_register_script(
+		'wp-block-serialization-spec-parser',
+		gutenberg_url( 'build/block-serialization-spec-parser/index.js' ),
+		array(),
+		filemtime( gutenberg_dir_path() . 'build/block-serialization-spec-parser/index.js' ),
+		true
```

Comment:
Do we really need this `true` in all registered scripts? I feel it's only necessary for `edit-post`?

Reply:
I think it fixes some issues, I remember seeing PR from @pento which added them everywhere.